### DISCUSSION TITLE
1999 江西 农业税事件

### DISCUSSION THREAD

**Discussion by diaohancai** [WeNeedHome/SummaryOfLoanSuspension#862] [General]
1999年8月份，在[江西丰城]有一位周姓农民，他鉴于当时农民生活艰辛，不仅要劳作还要承受繁重的[农业税]，于是就自费整理了当时国家有关减轻农民负担的文件，并将其发散到每一个人的手里。同时他鼓励农民争取自己的利益，去抵制一些不合理的一些上缴费用。

    当时的乡政府得知之后，大概就以妖言惑众、扰乱民心的罪名将周抓到了所谓的学习站进行改进学习，问题却是仅在两天后，周就死亡了。

    得知消息的周氏家属纠结了50多人闹到了乡政府的门口要求给一个合理的说法，当时的乡政府却没有意识到问题的严重性，继续敷衍了事，把这些人赶出了乡政府。

    这种行为彻底惹恼了农民，好好的人不明不白地死，再联想到那些年乡政府为完成上级交代的任务不顾农民的死活，当时在基层农村干部中还流传着这样一个顺口溜，“上吊不解绳，喝药不接瓶，投河不拉人。要敢打人，就抓你的人。

    ”可见当时的村干部早已变质。 很快，这件事迅速在四个乡镇发酵。很快，总共四个乡镇的数万百姓拿上自家的农具，一大帮人浩浩荡荡地来到乡政府门口，，一心想着报复的农民已经被冲昏了头脑，他们冲进乡政府，把乡长和一名乡干部从二楼扔下，随即就挖出了一个很大的洞，将两人活埋。而前来调和的派出所所长和一名民警也被当场打死，派出所所长的尸体被吊在树上示众。

    见到此情况的乡党委书记乘乡中学一教师的摩托车跑到县城，当夜，江西省调派大批警力进驻丰城，武装镇压，这件事才逐渐平息。

    不久之后，国务院召开紧急电视电话会议，乡镇一级的两个正职全部参加，会上通报了多起因农民负担过而死人的重大群体事件，鉴于事态的严重性，会上强调了全国立即停止以强制手段征收上缴农业税。

    2000年3月2日，我国发布了关于进行农村税费改革试点工作的通知，并且正式在江西开始了一个农村税费改革的试点。3年后，国务院再次发表通知，全面推进农村税费改革试点工作并在全国推广开来。

    2005年上半年，中国的22个省市已经免征农业税，2005年底有28个省，大部分的县和市都全面免征了农业税。2006年1月1日，中国彻底废止了《农业税条例》，从此在中国延续两千多年农业税正式成为历史。而这一切一定程度上确实是归功于[江西丰城事件]，让国家看到了农民的困境和切确的需求。

作者：匿名用户
链接：https://www.zhihu.com/question/278439158/answer/2479176429
来源：知乎
著作权归作者所有。商业转载请联系作者获得授权，非商业转载请注明出处。

**Comment by xiaer93** [comment]
哪有什么岁月静好，只不过有人替你寻衅滋事。

**Comment by Accelerator6666** [comment]
维稳这么多年，血性没剩多少了。

**Comment by WE-color** [comment]
CPC欠着农民很多，他们以为我们新一代人都忘了，说什么先富带后富都是画大饼，但凡有带你脑子的都记着呢，现代社会信息流通不要太容易

**Comment by TALK1949** [comment]
这事当年吹了好久，后来我一查台湾早就取消了，而且之前的农业税分摊到其他税种里面了。

**Comment by xiejun946** [comment]
好像从古至今，中国这种儒家文化，就是滞后性，先要闹出一个什么事情，才能给个紧急补救办法。这么说，非要搞出一个天安门事件，才有可能解决

**Reply by xiazhifengtest** [reply]
这个事情我听过但是具体是啥能另开个话题说么

**Comment by k624035508** [comment]
没这件事之前  已经开始试点取消农业税改革了
